### DISCUSSION TITLE
HOWTOBUILD 문서 업데이트 논의

### DISCUSSION THREAD

**Discussion by yeonns** [euphony-io/euphony#64] [General]
#56 #61 반영으로 인해, build와 test 관련 스크립트가 수정되어 HOWTOBUILD 문서도 업데이트되어야 합니다 😄 
(issue: #62)
@seoh02h 님이 초안을 작성해주시면 추가/수정할 내용이나 영어 번역등에 대해 다같이 피드백해주시면 좋을거같아요!

**Comment by designe** [comment]
이번 네이티브 유닛 테스트에 대한 부분은 블로깅처럼 정리해봐도 좋겠다는 생각이 듭니다.
여기 디스커션에 추가하면 좋을 것 같은 asset이나 내용들이 있으면 채워주세요.

제가 초안은 작성해볼게요 :)

안드로이드 네이티브 유닛테스트의 기본부터 Euphony에 적용된 유닛테스트가 얼마나 편리한지에 대한 글을 써보도록 하겠습니다 ㅋㅋㅋ

**Reply by designe** [reply]
아 혹시라도 @seoh02h 님이나 다른분이 작성하고 싶으시다면! 양보해드릴 수 있습니다 ㅋㅋㅋㅋㅋㅋㅋ

**Reply by seohy02** [reply]
ㅎㅎㅎ 멘토님이 써주시면 더욱 좋을 것 같아요!!! 감사합니다!
